plugins/circleci/src/components/PipelinesPage/lib/Pipelines/Pipelines.test.tsx

```diff
@@ -13,26 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import React from 'react';
-import { CITable } from '../CITable';
-import { useBuilds } from '../../../../state';
+import { Pipelines } from './Pipelines';
+import { renderInTestApp } from '@backstage/test-utils';
+
+jest.mock('../CITable', () => ({
+  CITable: () => '<CITable />',
+}));
+
+jest.mock('../../../../hooks/usePipelines', () => ({
+  usePipelines: jest.fn().mockReturnValue([{ loading: false, value: [] }, {}]),
```

Comment:
this is a pretty "deep" and implementation dependent mock, maybe supply a mock of the actual API in a wrapper instead?

Reply:
Fixed